README.md

```diff
@@ -11,7 +11,7 @@ If you want to contribute or need help getting started, join us on Slack by fill
 To develop this extension the following environment is used
```

Comment:
I updated from gulp build/flow to npm start/build. But, needs a couple more updates.